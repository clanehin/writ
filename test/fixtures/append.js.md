```js
//:: functions :://
```

```js
//== functions ==//
function one() { return 'one'; }
```

```js
//== functions ==//
function two() { return 'two'; }
```

```js
//== functions ==//
function three() { return 'three'; }
```
