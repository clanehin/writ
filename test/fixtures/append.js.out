function one() { return 'one'; }
function two() { return 'two'; }
function three() { return 'three'; }
